index.js

```diff
@@ -1,64 +1,68 @@
 'use strict';
 
-var assign = require('object.assign');
+const { declare } = require('@babel/helper-plugin-utils');
 
-var modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
-  strictMode: true, // add "use strict"
-  strict: false // this allows __esModule to be exported
-}];
-
-var defaultTargets = {
+const defaultTargets = {
   android: 30,
   chrome: 35,
   edge: 14,
   explorer: 9,
   firefox: 52,
   safari: 8,
-  ucandroid: 1
+  ucandroid: 1,
 };
 
-function buildTargets(options) {
-  return assign({}, defaultTargets, options.additionalTargets);
+function buildTargets({ additionalTargets }) {
+  return Object.assign({}, defaultTargets, additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
-  var transpileTargets = (options && options.targets) || buildTargets(options || {});
```

Comment:
I removed this `transpileTargets` renaming to keep the destructure and assignment neater. If it's important I'll add it back.